web/public/locales/de.json

```diff
@@ -1644,135 +1644,135 @@
     },
     "US-NW-CHPD": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "PUD Nr.1 von Chelan County"
+      "zoneName": "Public Utility District Nr.1 von Chelan County"
     },
     "US-NW-DOPD": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "PUD Nr.1 von Douglas County"
+      "zoneName": "Public Utility District Nr.1 von Douglas County"
     },
     "US-NW-GCPD": {
       "countryName": "Vereinigte Staaten",
-      "zoneName": "PUD Nr.2 von Grant County, Washington"
+      "zoneName": "Public Utility District Nr.2 von Grant County, Washington"
```

Comment:
These still have "von" in them, should they?

Reply:
I left those in on purpose. My thinking was that not many people know what a `Public Utility District` is, so I thought, *This is the PUD Nr. 2 of Grant County etc.* (in german *Das ist das Public Utility Disctrict Nummer 2 von Grant County*) makes more sense